### PR TITLE
feat: add alert channel schemas and new flags to deploy

### PR DESCRIPTION
**Pull request by ianaya89**
I hereby confirm that I followed the code guidelines found at [engineering guidelines](https://www.notion.so/checkly/Engineering-Guidelines-a3a165a203a04dc1a112f0e26b2f2d3f)

## ⚙️ Affected Components

- [x] Commands
- [x] Modules
- [x] Services
- [ ] SDK
- [ ] Tests

### 📝 Notes for the Reviewer
- Add local alert channel schema
- Add new flags to deploy command

### CODE REVIEW

File: src/parser/resource-parser.js [hunk 1]
```diff
@@ -176,18 +173,16 @@ function parseAlertChannelSubscriptions (
 ) {
   const alertChannelSubscriptions = {}
 
-  if (resource.alertChannelSubscriptions) {
-    resource.alertChannelSubscriptions.forEach((subscription) => {
-      const { name, activated } = subscription
-      const logicalId = `${resourceLogicalId}/${name}`
+  resource.alertChannelSubscriptions.forEach((subscription) => {
```
Comment (umutuzgur) [inline src/parser/resource-parser.js:176]: This can no longer be null?

File: src/parser/resource-parser.js [hunk 2]
```diff
@@ -196,12 +191,16 @@ function parseAlertChannelSubscriptionsTree (tree, type = 'check') {
...
-      ...parseAlertChannelSubscriptions(tree[key], key, type)
+    if (tree[key].alertChannelSubscriptions?.length) {
+      parsedTree = {
+        ...parsedTree.alertChannelSubscriptions,
+        ...parseAlertChannelSubscriptions(tree[key], key, type)
+      }
     }
   })
 
+  // console.log(parsedTree)
```
Comment (umutuzgur) [inline src/parser/resource-parser.js:202]: leftover I think

File: src/parser/resource-parser.js (hunk 1 above)
Comment (ianaya89) [reply src/parser/resource-parser.js:176]: I moved the validation to the upper function `parseAlertChannelSubscriptionsTree` but when using schema at parser we always get a value (empty array as default).

**Review by umutuzgur — APPROVED** [review]
LGTM

**Pull request merged**